products/java.md

```diff
@@ -125,6 +125,15 @@ releases:
     releaseDate: 2006-12-12
     link: https://www.oracle.com/java/technologies/javase/6-relnotes.html#R160_211
     latestReleaseDate: 2018-10-16
+    
+-   releaseCycle: "5"
+    lts: false
+    support: 2015-04-01
+    eol: 2009-11-01
+    latest: "1_5_0_22"
```

Comment:
Latest from
https://github.com/endoflife-date/release-data/blob/b197fef895f13fc9fce100325a7d0c16314a98c6/releases/java.json#L161
> "5.0u85": "2015-04-14",

Also here: https://www.java.com/releases/ -> search for `5.0u85`

Reply:
Done